lib/msal-common/src/scripts/metadata.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const axios = require("axios");
```

Comment:
Can we move this to `lib/msal-common/scripts`?

Reply:
I have moved this scripts to the proposed location.